default.hbs

```diff
@@ -12,7 +12,13 @@
     {{!-- Theme assets - use the {asset} helper to reference styles & scripts,
     this will take care of caching and cache-busting automatically --}}
     <link rel="stylesheet" type="text/css" href="{{asset "built/screen.css"}}" />
-
+    <script src="https://twitter.github.io/typeahead.js/js/handlebars.js"></script>
+    <script>
+        Handlebars.registerHelper("trimString", function(passedString) {
+        var theString = passedString.substring(1,5);
+        return new Handlebars.SafeString(theString)
```

Comment:
can you stick some logging to the console in here?

```js
 Handlebars.registerHelper("trimString", function(passedString) {
 console.log("passed string: " + passedString);
 var theString = passedString.substring(1,5);
 console.log("theString: " + theString);
 return new Handlebars.SafeString(theString)
```

Reply:
@justinsantoro 6a6cb071cfd2068c737af8c8c428c023d2ec55b4

Reply:
what do we get